### ISSUE TITLE
Why don't mask during Testing?

### ISSUE THREAD

**Issue by sev777** [kssteven418/LTP#10]
https://github.com/kssteven418/LTP/blob/f1d5ec88aba913de5e2b4aa502af9cf0ab7bb13f/src/transformers/models/ltp/modeling_ltp.py#L247

        if self.training and not self.hard_masking:
            if pruner_outputs is not None:
                threshold, pruning_scores = pruner_outputs['threshold'], pruner_outputs['scores']
                self.mask = torch.sigmoid((pruning_scores - threshold) / self.temperature)
                layer_output = layer_output * self.mask.unsqueeze(-1)
